packages/client/components/PokerCard.tsx

```diff
@@ -51,6 +51,17 @@ interface CardBaseProps {
   isSelected: boolean,
   totalCards: number
 }
+
+// const getRotation = (totalCards: number, idx: number) => {
```

Comment:
coming back to this, i think a _very_ subtle arch in the hand will look good & allow us to fit in an extra card or 2